src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +343,40 @@ Demos
 
 ## FAQ
```

Comment:
Definitely want to leave these. Most are preemptive for this type of announcement. And the new ones are coming from what people are asking on hacker news.